policy/release/slsa_provenance_available.rego

```diff
@@ -1,16 +1,16 @@
 #
 # METADATA
-# title: SLSA - Available
+# title: SLSA - Provenance - Available
 # description: |-
-#   The SLSA Available requirement states the following:
+#   The SLSA Provenance Available requirement states the following:
 #
 #   "The provenance is available to the consumer in a format that the consumer accepts. The
 #   format SHOULD be in-toto SLSA Provenance, but another format MAY be used if both producer
 #   and consumer agree and it meets all the other requirements."
 #
 #   This package only accepts the in-toto SLSA Provenance format.
 #
-package policy.release.slsa_available
```

Comment:
Does the `slsa` collection need to be updated to reflect the new package name?

Reply:
🤦

Fixed it